packages/kdot/lib/fwd.js

```diff
@@ -6,26 +6,53 @@ import { core, pfwd } from './k8sApi.js'
 
 const logger = createLogger({ namespace: 'kdot', level: 'info' })
 
+async function getPod (namespace, name) {
+  // FIXME: Maybe we can implement our own local load balancer to simulate
+  // the service and distribute traffic to all of the pods instead of just
+  // the first one?
+  const { body: { items: [pod] } } = await core.listNamespacedPod(
+    namespace,
+    undefined,
+    undefined,
+    undefined,
+    undefined,
+    `app=${name}`
+  )
```

Comment:
Should check to see if any of the pods are running and return one instead of just the first one.